controllers/greptimedbcluster/controller.go

```diff
@@ -298,3 +307,25 @@ func (r *Reconciler) setObservedGeneration(cluster *v1alpha1.GreptimeDBCluster)
 		cluster.Status.ObservedGeneration = &generation
 	}
 }
+
+func (r *Reconciler) removeMonitoringDB(ctx context.Context, cluster *v1alpha1.GreptimeDBCluster) error {
+	objects := map[string]client.Object{
+		common.MonitoringServiceName(cluster.Name): new(v1alpha1.GreptimeDBStandalone),
```

Comment:
question：Why the greptimedbcluster type need to remove the monitoring of greptimedbstandalone?

Reply:
When the `.spec.monitoring.enabled` is `false`, we need to clean up.